### DISCUSSION TITLE
Supabase email change confirmation error

### DISCUSSION THREAD

**Discussion by NielsVeen** [supabase/supabase#19015] [Questions] [answered]
I'm currently having some issues with the email change flow in supabase.js (v2.33.2). 
When a user changes email and clicks the confirmation link the server throws an error:
`https://mydomain.com?error=server_error&error_code=500&error_description=Error+confirm+email#error=server_error&error_code=500&error_description=Error+confirm+email`

Does anyone have any why this might happen? Can't find any context on this in the docs.

**Comment by GaryAustin1** [comment]
You should check your auth and Postgres logs in the dashboard and see if there are any errors.

**Reply by NielsVeen** [reply]
I'm seeing this: permission denied for table users

**Comment by NielsVeen** [comment, marked as answer]
The problem was that there was a function updating my public.users table when the auth.users table was updated. The RLS Policy on this public table was preventing the update, resulting in an error. Updating this function to use Security Definer solved it for me.
